src/executables/csql.c

```diff
@@ -1806,6 +1842,13 @@ csql_execute_statements (const CSQL_ARGUMENT * csql_arg, int type, const void *s
       logddl_set_csql_input_type (CSQL_INPUT_TYPE_EDITOR);
     }
 
+  strcpy (prompt_cpy, stmts);
```

Comment:
stmts may be larger than 100 bytes.

Reply:
Right. Its size expands till the memory lack.

I should consider about it. Thank you.

Reply:
add_history is needed to be fixed.